homeassistant/components/plugwise/sensor.py

```diff
@@ -290,19 +290,18 @@ async def async_setup_entry(
     coordinator = hass.data[DOMAIN][config_entry.entry_id][COORDINATOR]
 
     entities: list[PlugwiseSensorEnity] = []
-    for device_id, device_properties in coordinator.data.devices.items():
+    for device_id, device in coordinator.data.devices.items():
         for description in SENSORS:
             if (
-                "sensors" not in device_properties
-                or device_properties["sensors"].get(description.key) is None
+                "sensors" not in device
+                or device["sensors"].get(description.key) is None
```

Comment:
Earlier commit to blame, but if `sensors` is missing in the dict this would create an Exception?

Reply:
no? If `"sensors" not in device` = `True`, the second part of the statement isn't evaluated?

Reply:
🆗